transformer_engine/pytorch/ops/op.py

```diff
@@ -260,6 +275,21 @@ def _maybe_update_fp8_meta(cls, fp8_meta: Optional[dict[str, Any]]) -> None:
                         pad=(0, 0, 0, amax_history_len - curr_len),
                     )
 
+            # Update global buffers for amax reductions
```

Comment:
This does not look like operation API specific thing - was the lack of this in the previous code a bug?

Reply:
Yep, if the amax history length changes then I don't expect amax reductions to be handled correctly.